ios/FluentUI/AvatarGroup/AvatarGroup.swift

```diff
@@ -141,10 +151,14 @@ public struct AvatarGroup: View, ConfigurableTokenizedControl {
                     let nextAvatarHasRing = !isLastDisplayed ? avatars[nextIndex].isRingVisible : false
 
                     // Calculating the different interspace scenarios considering rings.
+                    let currentTokens = avatarView.tokens
+                    let ringOuterGap = currentTokens.ringOuterGap
+                    let ringOffset = currentTokens.ringInnerGap + currentTokens.ringThickness + ringOuterGap
                     let stackPadding = interspace - (currentAvatarHasRing ? ringOffset : 0) - (nextAvatarHasRing ? ringOuterGap : 0)
 
                     // Finalized calculations for x and y coordinates of the Avatar if it needs a cutout, including RTL.
-                    let cutoutSize = isLastDisplayed ? ringGapOffset + imageSize : nextAvatarSize
+                    let cutoutSize = isLastDisplayed ? overflowAvatar.totalSize : nextAvatarSize
```

Comment:
Is it possible to get rid of he cutoutSize constant and use the nextAvatarSize only?
It can have either the next avatar's totalSize or just the overflow avatar's totalSize depending on the scenario.